from spaceone.core.error import *


class ERROR_CONNECTOR_CALL_API(ERROR_UNKNOWN):
    _message = 'API Call Error: {reason}'
